### PR TITLE
[CLOUD-2798] - Update to EAP CD 14

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-2798
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ schema_version: 1
 
 name: "jboss-eap-7-tech-preview/eap-cd-openshift"
 description: "The JBoss EAP continuous delivery (JBoss EAP CD) releases are intended to be Technology Preview - The JBoss EAP CD OpenShift container image is provided as technology preview. It is intended for development use only. It should NOT be deployed on production or in environments that are not intended for development use."
-version: "13.0"
-from: "jboss-eap-7/eap-cd:13.0"
+version: "14.0"
+from: "jboss-eap-7/eap-cd:14.0"
```
Comment (bstansberry) [inline image.yaml:6]: I'll change https://github.com/jboss-container-images/jboss-eap-7-image/pull/93 so image.yaml there says 14.0.

File: templates/eap-cd-amq-s2i.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     },
     "labels": {
         "template": "eap-cd-amq-s2i",
-        "xpaas": "1.5.0"
+        "xpaas": "14.0"
```
Comment (bstansberry) [inline templates/eap-cd-amq-s2i.json:20]: IIRC the 'version' field in 'metadata' : 'annotations' above is supposed to match this. 

This same comment applies to all the templates.

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: image.yaml (hunk 1 above)
Comment (bstansberry) [reply image.yaml:6]: Done

**Comment by luck3y** [comment]
@bstansberry I think this should be g2g now.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**